hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieStreamingSink.scala

```diff
@@ -107,6 +107,8 @@ class HoodieStreamingSink(sqlContext: SQLContext,
       return
     }
 
+    validateMultiWriterConfigs(options)
```

Comment:
can we avoid validation per invocation? is there an init method hook? or a common place where we already run some writer config validation.